Uplift/Views/GymDetail/Facilities/PriceInformationCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  PriceInformationCell.swift
+//  Uplift
+//
+//  Created by Yana Sang on 10/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class PriceInformationCell: UICollectionViewCell {
+
+    struct Constants {
+        static let lineHeight: CGFloat = 20
+        static let padding: CGFloat = 12
+    }
+
+    // MARK: - Private view vars
+    private let background = UIView()
+    private let itemsList = UITextView()
+    private let pricesList = UITextView()
+
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func configure(items: [String], prices: [String]) {
+        itemsList.attributedText = generateAttributedString(for: items, font: ._14MontserratRegular)
+        pricesList.attributedText = generateAttributedString(for: prices, font: ._14MontserratSemiBold, alignment: .right)
+
+        pricesList.snp.remakeConstraints { make in
+            make.top.equalTo(background).offset(Constants.padding)
+            make.trailing.bottom.equalTo(background).inset(Constants.padding)
+            make.width.equalTo(pricesList.intrinsicContentSize.width)
+        }
```

Comment:
How come theres a width constraint in this but not in the `setupConstraints`?

Reply:
Also same comment about `make.top.trailing.bottom.equalTo(background).inset(Constants.padding)`

Reply:
Also can we move this to its own function since its repeated code? `remakeConstraints()`, i think, will have the same effect as `makeConstraints()` if there have been no constraints previously made for that view.

Reply:
would this still work if we just removed the `makeConstraints` code in `setupConstraints` and just changed this to `makeConstraints`?

Reply:
Idts cuz the cell reuse would screw it over by trying to call `makeConstraints` on a view that already has constraints. This would especially get rekt with the bottom and width constraints. Might wanna test tho but im like 90% sure

Reply:
oh you're right

Reply:
it looks okay? i.e. like not doing any constraints on `prices` initially and only setting them in the configure. but that seems like a bad idea

Reply:
on second thought i think we can just remove `makeConstraints` in setupConstraints and just have this `remakeConstraints`

Reply:
Do we also need to do this for `itemsList`?? Seems like both of their constraints should be set in this function and not in `setupConstraints`